README.md

```diff
@@ -54,8 +54,8 @@ You can also use a named volume which will be created with the correct ownership
 ```text
 # create <working directory>/zen-data with correct ownership
 docker run --rm -v "$(pwd)/zen-data:/data" ubuntu:18.04 bash -c 'chown -R nobody:nogroup /data'
-# start rosetta-zen
-docker run -d --rm --ulimit "nofile=100000:100000" -v "$(pwd)/zen-data:/data" -e "MODE=ONLINE" -e "NETWORK=MAINNET" -e "PORT=8080" -p 8080:8080 -p 9033:9033 rosetta-zen:latest
+# start rosetta-zen. Zend version needs to be specified (ex. v3.2.0)
+docker run -d --rm --ulimit "nofile=100000:100000" -v "$(pwd)/zen-data:/data" -e "MODE=ONLINE" -e "NETWORK=MAINNET" -e "NODE_VERSION=${ZEND_VERSION}" -e "PORT=8080" -p 8080:8080 -p 9033:9033 rosetta-zen:latest
```

Comment:
Why do we need to specify `NODE_VERSION`? I can't see it being used anywhere. Zend version could also be detected on the fly in entypoint.sh by running `zend -version | head -n1 | cut -d " " -f 4`, or storing the var from the Makefile via ENV NODE_VERSION=$ARG in Dockerfile.